packages/cli-config/package.json

```diff
@@ -11,7 +11,7 @@
     "@react-native-community/cli-tools": "^10.1.1",
     "chalk": "^4.1.2",
     "cosmiconfig": "^5.1.0",
-    "deepmerge": "^3.2.0",
+    "deepmerge": "^4.3.0",
```

Comment:
this turned out to be unnecessary, but checking the release notes the only breaking change is umd -> cjs, which is fine for us